database/fakerData.js

```diff
@@ -1,11 +1,156 @@
 var faker = require('faker');
 
-var user = {
-  experience_id: faker.random.number({min:1, max:100});
-  username: faker.name.firstName(),
-  user_id:
-  date: faker.date.past(2, '2020-02-06');
-  stars: faker.random.number({min:0, max:5});
-  review: faker.lorem.sentence(),
-  image: faker.image.avatar()
-};
+//need at least 2500 reviews.  can do 200/month for the last 13 months
+module.exports.createFakeData = () => {
+  let results = [];
+  for (var i = 0; i <= 2600; i++) {
+    if (i < 201) {
+      var user = {
+        experience_id: faker.random.number({min:1, max:100}),
+        username: faker.name.firstName(),
+        avatar: faker.image.avatar(),
+        date: '20-02-01',
+        stars: faker.random.number({min:0, max:5}),
+        review: faker.lorem.sentence()
+      };
+      results.push(user);
+    }
+    if (i >= 200 && i < 401) {
```

Comment:
The first if statement goes from 0 to 200. Would this if statement go from 201 to 400 so something like if i > 200 && i < 401?

Reply:
I have fixed this and it will be in my next commit / PR